app/infrastructure/exceptions/client_exception.py

```
@@ -0,0 +1,20 @@
+from typing import Optional
+from bs4 import BeautifulSoup
+
+from app.infrastructure.exceptions.base_exception import BaseException
+from app.infrastructure.exceptions.error_type import ErrorType
+
+
+class ClientException(BaseException):
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)